.travis.yml

```diff
@@ -10,3 +10,14 @@ python:
   - "nightly" # currently points to 3.6-dev
 # command to run tests
 script: nosetests
+notifications:
+  email:
+    on_success: never
+    on_failure: always
+deploy:
+  provider: pypi
+  user: bosswissam
+  password:
+    secure: 
+ on:
+    branch: master
```

Comment:
It shouldn't be triggered for every single commit in master. Only tagged builds should publish dist.